docs/README.md

```diff
@@ -1,177 +1,69 @@
-| `footerShippingPriceLabelContainer` | The shipping price label container                                               | [MinicartFooter](/react/components/MinicartFooter.js)   |
-| `sidebarRightCaretContainer`        | The caret icon container in the sidebar variant of the minicart                  | [Sidebar](/react/components/Sidebar.js)                 |
-| `sidebarItemQuantityContainer`      | The item quantity container in the sidebar variant of the minicart               | [Sidebar](/react/components/Sidebar.js)                 |
-
-## Troubleshooting
-
-You can check if others are passing through similar issues [here](https://github.com/vtex-apps/minicart/issues). Also feel free to [open issues](https://github.com/vtex-apps/minicart/issues/new) or contribute with pull requests.
-
-## Contributing
-
-Check it out [how to contribute](https://github.com/vtex-apps/awesome-io#contributing) with this project.
-
-## Tests
-
-To execute our tests go to `react/` folder and run `yarn test`
-
-### Travis CI
-
-[![Build Status](https://api.travis-ci.org/vtex-apps/minicart.svg?branch=master)](https://travis-ci.org/vtex-apps/minicart)
+| Prop name              | Type      | Description                                                                                              | Default value |
+| ---------------------- | --------- | -------------------------------------------------------------------------------------------------------- | ------------- |
+| `variation`            | `Enum`    | Define Minicart variation. (values: `'popup'`, `'drawer'` or `'link'`)                                   | `'sidebar'`   |
+| `openOnHover`          | `Boolean` | Whether the popup minicart should open when the user hovers over it.                                     | false         |
+| `linkVariationUrl`     | `String`  | The link associated to the `'link'` variation of the Minicart.                                           | false         |
+| `drawerSlideDirection` | `Enum`    | Controls the slide direction for the `'sidebar'` variation. (values: `'rightToLeft'` or `'leftToRight'`) | `rightToLeft` |
+| `maxDrawerWidth`       | `Number`  | The maximum width in pixels for the open sidebar Minicart.                                               | `440`         |
+
+## Customization
+
+| CSS Handles                     |
+| ------------------------------- |
+| `minicartWrapperContainer`      |
+| `minicartContainer`             |
+| `minicartContent`               |
+| `minicartFooter`                |
+| `minicartSideBarContentWrapper` |
+| `minicartIconContainer`         |
+| `minicartQuantityBadge`         |
+| `popupWrapper`                  |
+| `popupContentContainer`         |
+| `arrowUp`                       |
+| `popupChildrenContainer`        |
+
+## Deprecation notice
+
+The `minicart` interface has been deprecated in favor of this new `minicart.v2`, which is flexible.
```

Comment:
Do people know what "flexible" means?

Reply:
Hahahah I think they do

Reply:
Yep! I agree with Iaron. I believe this sentence should be something like: "Minicart block has been deprecated in favor of Minicart v2 which uses the Flex-Layout. If you’re still using the former version, you can find its documentation here: https://github.com/vtex-apps/minicart/blob/383d7bbd3295f06d1b5854a0add561a872e1515c/docs/README.md" 

Also, it should be stated before the whole documentation, below the block name. For example: 

- disclaimer 
- #Minicart 
- Sentence with an :warning: emoji before it. 

What do you guys think?

Reply:
Ok, maybe it is confusing hahahah
This `minicart` is not exactly flexible because of `flex-layout`, but because it can be structured from other blocks instead of being a single big block

Reply:
The blocks the user is going to use are the ones from `checkout-summary` and `product-list`